lib/parser/wat-modules.ts

```diff
@@ -287,6 +327,7 @@ export const module_: Parser<Module> = do_(($) => {
 					function_,
 					import_,
 					memory_,
+					table_,
```

Comment:
Trailing underscore `_` is to avoid conflicting with JavaScript words. `table` doesn't need underscore.